gui/qt/console.py

```diff
@@ -87,6 +87,8 @@ def __init__(self, parent):
         hbox_layout = QtWidgets.QHBoxLayout()
         layout.addLayout(hbox_layout)
 
+        hbox_layout.addSpacerItem(QtWidgets.QSpacerItem(16, 0, QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed))
```

Comment:
👍  for using the same constant.  Might want to comment here that it should be the same as the corner radius just so in the future people know ...

Reply:
I now separated the stylesheet into a color part, a common part and a border radius part and combine them at runtime. The border radius is a constant used to build the border radius style and to build the spacer items.